changelogs/fragments/10424-scaleway-update-zones.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway_* modules - update available zones and API URLs (https://github.com/ansible-collections/community.general/issues/10383, https://github.com/ansible-collections/community.general/pull/10424).
```

Comment:
Sorry, one last change request - we missed mentioning the inventory plugin in the changelog:
```suggestion
  - scaleway_* modules, scaleway inventory plugin - update available zones and API URLs (https://github.com/ansible-collections/community.general/issues/10383, https://github.com/ansible-collections/community.general/pull/10424).
```